code/obj/machinery/phone/phone.dm

```diff
@@ -337,6 +337,7 @@ TYPEINFO(/obj/machinery/phone)
 	var/obj/machinery/phone/parent = null
 	flags = TALK_INTO_HAND
 	w_class = W_CLASS_TINY
+	var/icon/handset_icon = null
```

Comment:
Seems to be unused?

Reply:
set in New on 353, and to compile the phone identifier segment for chat on 396